docs/semgrep-ci/ci-environment-variables.md

```diff
@@ -230,6 +232,59 @@ jobs:
 
 ```
 
+### `SEMGREP_REPO_DISPLAY_NAME`
+
+Set `SEMGREP_REPO_DISPLAY_NAME` to define the name used for the project associated with a given Semgrep scan. By default, `SEMGREP_REPO_DISPLAY_NAME` has the same value as `SEMGREP_REPO_NAME`. This allows you to use a different name for your project than the repository name or to scan a monorepo as multiple projects.
```

Comment:
A 'scan' sounds a little ambiguous? It could mean a discrete, specific scan that has been run, it could be a CLI scan, etc. Setting `SEMGREP_REPO_DISPLAY_NAME` essentially creates a separate project entry, so maybe we can say:
```suggestion
Set `SEMGREP_REPO_DISPLAY_NAME` to define the name used for the project in Semgrep Cloud Platform. By default, `SEMGREP_REPO_DISPLAY_NAME` has the same value as `SEMGREP_REPO_NAME`. This allows you to use a different name for your project than the repository name or to scan a monorepo as multiple projects.
```